core/storage/feedback/gae_models.py

```diff
@@ -260,11 +260,15 @@ class SuggestionModel(base_models.BaseModel):
     exploration_version = ndb.IntegerProperty(required=True, indexed=True)
     # Name of the corresponding state.
     state_name = ndb.StringProperty(required=True, indexed=True)
+    # Learner-provided description of suggestion changes.
+    description = ndb.StringProperty(required=True, indexed=True)
```

Comment:
Sorry, I missed this -- can you change this to TextProperty and make it "indexed=False"? StringProperty takes max 500 chars.